src/main/markdown/makinggwtbetter.md

```diff
@@ -538,6 +538,12 @@ Finally, to actually create an issue is slightly more involved, but most of the
 
 If you have any problems you might want to consult the gerrit documentation on [Uploading Changes](https://gerrit-review.googlesource.com/Documentation/user-upload.html) 
 
+### Gerrit code review<a id="gerritcodereview"></a>
+
+Code review is an essential part of contribution workflow :
```

Comment:
I love that sentence! :smiley: 

The following paragraph needs some work though. Let me think a bit about it, I'll try to propose a new wording.

Good work on the heading and where to put that section too.